crates/node/src/networking/p2p/pea2pea.rs

```diff
@@ -49,6 +50,8 @@ pub struct P2P {
 
     // Send Tx to the process loop.
     tx_sender: TxEventSender<P2pSender>,
+
+    protocol_version: u64,
```

Comment:
Although the value of this is constant (at the moment), I moved it to member variable so that its value can be shared / used across the code in uniform way and it is changed only from one place.